### DISCUSSION TITLE
What's the easiest way to create a contact form?

### DISCUSSION THREAD

**Discussion by SamGreenwood1** [nunocoracao/blowfish#524] [Q&A]

**Comment by nunocoracao** [comment]
Not an issue moving into discussions

**Comment by nunocoracao** [comment]
The question is very open. I would say you can extend the theme with an override that includes the contact form. As to what that contact form is that will depend on what you want to achieve - e.g. mail button VS mailchimp integration

**Comment by JOduMonT** [comment]
@SamGreenwood1 personally I like [Tally.so](https://tally.so/)
you could also make your own choice between these [alternative](https://alternativeto.net/software/tally-forms/?feature=form-builder) ;)

**Comment by johndo100** [comment]
Google Sheets is a free solution.
I did it for my sites, will add to the theme soon when I have free time.
https://github.com/dwyl/learn-to-send-email-via-google-script-html-no-server

**Comment by sansubr** [comment]
Is there a way to add tailwind forms to this theme to add simple forms that can be used with like Netlify?

**Reply by nunocoracao** [reply]
@sansubr not natively in Blowfish, but Hugo allows you to for sure you just need to check their docs directly.

**Comment by epsilonhalbe** [comment]
I'm thinking about adding welcomments to my site, are you interested in getting the form elements I'll be doing back as a PR?
